core-services/src/main/java/org/zalando/nakadi/service/publishing/EventMetadata.java

```diff
@@ -1,25 +1,155 @@
 package org.zalando.nakadi.service.publishing;
 
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.generic.GenericRecordBuilder;
 import org.json.JSONObject;
 import org.springframework.stereotype.Component;
 import org.zalando.nakadi.util.FlowIdUtils;
 import org.zalando.nakadi.util.UUIDGenerator;
 
 import java.time.Instant;
+import java.util.UUID;
 
 @Component
 public class EventMetadata {
+    public static final String METADATA_FIELD = "metadata";
```

Comment:
This seems to be out of place.  The rest of field names here are for the metadata object itself, but this one is for the object containing the metadata...

And I'm not sure that we really need these as named constants in the first place (we have tests to verify correctness anyway). ;)

Reply:
I would like to keep this as static fields. 
This prevents any future update in one place and not the other.